beetsplug/fetchart.py

```diff
@@ -975,6 +974,26 @@ def get(self, album, plugin, paths):
             self._log.debug('Spotify: error loading response: {}'
                             .format(response.text))
             return
+
+
+class CoverArtUrl(RemoteArtSource):
+    NAME = "Cover Art URL"
+
+    def get(self, album, plugin, paths):
+        image_url = None
+        try:
+            image_url = album.items().get().cover_art_url
+            self._log.debug(f'Cover art URL {image_url} found for {album}')
+        except (AttributeError, TypeError):
```

Comment:
I'm not entirely sure when a `TypeError` would occur here, but I'll take your word for it!